__tests__/db.test.js

```diff
@@ -181,5 +261,16 @@ describe('app', () => {
         expect(res.body).toEqual({status: 400, msg : 'Bad request'});
       })
     })
+    test('PATCH - api/articles/:article_id - responds with 200 & updated votes property', () => {
+      return request(app)
+      .patch('/api/articles/1')
+      .send({
+        inc_votes: 'hello'
+      })
+      .expect(400)
+      .then((res) => {
+        expect(res.body).toEqual({ status: 400, msg: 'Bad request'});
+      })
+    })
```

Comment:
You can add more error tests here - your path takes an article id, what could go wrong with that? you might have already tested them before but this is a separate endpoint and needs to be checked again 😅